src/test/regress/sql/partition_pruning.sql

```diff
@@ -973,4 +973,103 @@ where  trg.key1 = src.key1
 
 DROP TABLE t_part1;
 
+-- Tests of partition pruning for case when default partition was exchanged without validation
+-- and contains rows, that satisfy constraints of other partition
+
+-- simple delete test case
```

Comment:
... delete test case from default partition. `Delete` operation has not to perform partition pruning